### PR TITLE
commiting dagPickle session when the airflow tasks run --ship-dag --interactive command

### LINKED ISSUE #15748: airflow tasks run --ship-dag not able to generate pickeled dag
**Apache Airflow version**: 2.0.1

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): No

**Environment**:

- **Cloud provider or hardware configuration**: local machine
- **OS** (e.g. from /etc/os-release): 18.04.5 LTS (Bionic Beaver)
- **Kernel** (e.g. `uname -a`): wsl2

**What happened**:
Getting Pickled_id: None

```
root@12c7fd58e084:/opt/airflow# airflow tasks run example_bash_operator runme_0 now --ship-dag --interactive
[2021-05-09 13:11:33,247] {dagbag.py:487} INFO - Filling up the DagBag from /files/dags
Running <TaskInstance: example_bash_operator.runme_0 2021-05-09T13:11:31.788923+00:00 [None]> on host 12c7fd58e084
Pickled dag <DAG: example_bash_operator> as pickle_id: None
Sending to executor.
[2021-05-09 13:11:34,722] {base_executor.py:82} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-05-09T13:11:31.788923+00:00', '--local', '--pool', 'default_pool', '--subdir', '/opt/airflow/airflow/example_dags/example_bash_operator.py']
[2021-05-09 13:11:34,756] {local_executor.py:81} INFO - QueuedLocalWorker running ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-05-09T13:11:31.788923+00:00', '--local', '--pool', 'default_pool', '--subdir', '/opt/airflow/airflow/example_dags/example_bash_operator.py']
[2021-05-09 13:11:34,757] {local_executor.py:386} INFO - Shutting down LocalExecutor; waiting for running tasks to finish.  Signal again if you don't want to wait.
[2021-05-09 13:11:34,817] {dagbag.py:487} INFO - Filling up the DagBag from /opt/airflow/airflow/example_dags/example_bash_operator.py
Running <TaskInstance: example_bash_operator.runme_0 2021-05-09T13:11:31.788923+00:00 [None]> on host 12c7fd58e084
```

**What you expected to happen**:

Pickled_id should get generated 

```
Pickled dag <DAG: example_bash_operator> as pickle_id: None
```

**How to reproduce it**:
run below command from command line in airflow environment
```
airflow tasks run example_bash_operator runme_0 now --ship-dag --interactive
```
**Would like to submit PR for this issue**: YES

### PR DESCRIPTION
**Pull request by vikramcse**
commiting dagPickle session when the airflow tasks run --ship-dag --interactive command is executed

Fixes #15748

### CODE REVIEW

File: airflow/cli/commands/task_command.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def _run_task_by_executor(args, dag, ti):
             with create_session() as session:
                 pickle = DagPickle(dag)
                 session.add(pickle)
+                session.commit()
                 pickle_id = pickle.id
                 # TODO: This should be written to a log
                 print(f'Pickled dag {dag} as pickle_id: {pickle_id}')
```
Comment (ashb) [inline airflow/cli/commands/task_command.py]:
```suggestion
            pickle_id = pickle.id
            # TODO: This should be written to a log
            print(f'Pickled dag {dag} as pickle_id: {pickle_id}')
```

**Review by ashb — CHANGES_REQUESTED** [review]
The create_session() context automatically commits when it exits.

File: airflow/cli/commands/task_command.py (hunk 1 above)
Comment (vikramcse) [reply airflow/cli/commands/task_command.py]: @ashb Ahh! Thanks for your inputs! I made the changes as per your suggestions.

earlier I was referring to below code
https://github.com/apache/airflow/blob/3711a29e5e253ede4ab12efe5990e83240e8a9a9/airflow/jobs/backfill_job.py#L782-L785

**Review by ashb — APPROVED** [review]

**Comment by vikramcse** [comment]
hello @ashb
the ci build is failing, I am not sure why it is failing.

**Review by potiuk — APPROVED** [review]

**Pull request merged**